src/explore-education-statistics-admin/src/pages/publication/components/PublicationAddExistingUsersTab.tsx

```diff
@@ -0,0 +1,104 @@
+import WarningMessage from '@common/components/WarningMessage';
+import React from 'react';
+import releasePermissionService, {
+  ManageAccessPageContributor,
+} from '@admin/services/releasePermissionService';
+import { Formik } from 'formik';
+import { Form, FormFieldCheckboxGroup } from '@common/components/form';
+import Button from '@common/components/Button';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import { generatePath, useHistory } from 'react-router-dom';
+import { publicationManageTeamAccessReleaseRoute } from '@admin/routes/routes';
+import { ReleaseRouteParams } from '@admin/routes/releaseRoutes';
+import { Release } from '@admin/services/releaseService';
+
+interface AddExistingUsersFormValues {
+  userIds: string[];
+}
+
+export interface Props {
+  publicationId: string;
+  release: Release;
+  contributors: ManageAccessPageContributor[];
+}
+
+const PublicationAddExistingUsersTab = ({
+  publicationId,
+  release,
+  contributors,
+}: Props) => {
+  const history = useHistory();
+
+  const handleSubmit = useFormSubmit<AddExistingUsersFormValues>(
+    async values => {
+      await releasePermissionService.updateReleaseContributors(
+        release.id,
+        values.userIds,
+      );
+      history.push(
+        generatePath<ReleaseRouteParams>(
+          publicationManageTeamAccessReleaseRoute.path,
+          {
+            publicationId,
+            releaseId: release.id,
+          },
+        ),
+      );
+    },
+    [],
+  );
+
+  if (!contributors || !contributors.length) {
+    return (
+      <WarningMessage>
+        There are no contributors for this release's publication.
+      </WarningMessage>
+    );
+  }
+
+  const initialValues: AddExistingUsersFormValues = {
+    userIds: contributors
+      .filter(c => c.releaseRoleId !== undefined)
+      .map((c: ManageAccessPageContributor) => c.userId),
+  };
+
+  const checkboxOptions = contributors.map(
+    (contributor: ManageAccessPageContributor) => {
+      return {
+        label: `${contributor.userFullName} (${contributor.userEmail})`,
+        value: contributor.userId,
+      };
+    },
+  );
+
+  return (
+    <>
+      <h2>Add existing publication users to release ({release.title})</h2>
```

Comment:
Shouldn't need this second heading. The main page title should be something like: 'Manage release contributors (Academic Year 2020/21)'.

Consequently, we don't need the wrapping fragment either.